Library/Homebrew/os/mac/xcode.rb

```diff
@@ -359,8 +359,8 @@ def latest_clang_version
       sig { returns(String) }
       def minimum_version
         case MacOS.version
-        when "13" then "14.0.0"
-        when "12" then "13.0.0"
+        when "13" then "14.1.0"
+        when "12" then "13.1.0"
```

Comment:
I think `detect_clang_version` doesn't pick up the `.1` sadly because the clang version didn't change much.

Reply:
```suggestion
        when "13" then "14.1.0"
        when "12" then "13.1.0"
```